hack/examples/bases/capd/kustomization.yaml

```diff
@@ -0,0 +1,23 @@
+# Copyright 2023 D2iQ, Inc. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+configurations:
+- kustomizeconfig.yaml
+
+resources:
+- https://github.com/kubernetes-sigs/cluster-api/releases/download/v1.5.1/clusterclass-quick-start.yaml
+- https://github.com/kubernetes-sigs/cluster-api/releases/download/v1.5.1/cluster-template-development.yaml
```

Comment:
nit: it would be cool to get the version from the installed `clusterctl`

Reply:
Yes! But like I said, I don't know how to use variables in kustomizations :facepalm: